python/cudf/cudf/tests/groupby/test_agg.py

```diff
@@ -16,3 +16,13 @@ def test_agg_count_dtype(empty):
         df = df.iloc[:0]
     result = df.groupby("a").agg({"c": "count"})
     assert result["c"].dtype == np.dtype("int64")
+
+
+@pytest.mark.parametrize("attr", ["agg", "aggregate"])
+def test_series_agg(attr):
+    df = cudf.DataFrame({"a": [1, 2] * 2, "b": [0] * 4})
```

Comment:
Let's just spell it out since the data is short.
```suggestion
    df = cudf.DataFrame({"a": [1, 2, 1, 2], "b": [0, 0, 0, 0]})
```